src/components/a-icon/_template.js

```diff
@@ -7,8 +7,10 @@ export default ({
   icon,
   classes = 'a-icon',
   pathPrefix,
-} = {}) => {
-  const axaPLibConfig = window.__axaPLibConfig || {};
+} = {
```

Comment:
hmh, I don't really agree with this auto-fix from eslint. Empty objects should be fine to be on one line